hawkular-wildfly-monitor/src/main/assembly/module-assembly.xml

```diff
@@ -39,6 +39,14 @@
       <directoryMode>0755</directoryMode>
     </fileSet>
   </fileSets>
+  <files>
+    <file>
+      <source>${project.basedir}/src/main/assembly/subsystem.xml</source>
+      <outputDirectory>/${moduleDir}</outputDirectory>
+      <destName>subsystem-snippet.xml</destName>
+    </file>
+  </files>
+
 
   <dependencySets>
     <dependencySet>
```

Comment:
I think it won't cause issues as it is not referenced in module.xml, but it's maybe better to place it deeper into ZIP right next to module.xml